exasol/nb_connector/slc/script_language_container.py

```diff
@@ -79,10 +79,16 @@ def __init__(
         self.flavor = SlcFlavor(name).verify(secrets)
         self.compression_strategy = SlcCompressionStrategy(name).verify(secrets)
         self.workspace = Workspace.for_slc(name)
-        if not self.flavor_path.is_dir():
+        if not self.checkout_dir.is_dir():
             raise SlcError(
                 f"SLC Git repository not checked out to {self.checkout_dir}."
             )
+        if not self.flavor_path.is_dir():
+            raise SlcError(
+                f"Given flavor {self.flavor} not found in version {constants.SLC_RELEASE_TAG} of "
+                "Script-Languages-Release. "
+                "Check out available flavors at https://github.com/exasol/script-languages-release/releases/tag/{constants.SLC_RELEASE_TAG}.")
```

Comment:
@tkilias , @sgn4sangar  I'am just thinking: Should we simplify the list of available flavrs in the slc release? Currently the user can find this list by clicking [Package Version Comparison between Release 9.7.0 and 9.6.0](https://github.com/exasol/script-languages-release/blob/master/doc/changes/package_diffs/9.7.0/README.md). However, there the template and standard flavors are mixed....